src/rubrix/client/__init__.py

```diff
@@ -194,16 +220,22 @@ def load(
         if task == TaskType.TEXTCLASSIFICATION:
             from rubrix.sdk.api.text_classification import _get_dataset_data
 
-            map_fn = self._text_classification_sdk_to_record
+            map_fn = self._text_classification_sdk_to_client
             request_class = TextClassificationQuery
         elif task == TaskType.TOKENCLASSIFICATION:
             from rubrix.sdk.api.token_classification import _get_dataset_data
 
-            map_fn = self._token_classification_sdk_to_record
+            map_fn = self._token_classification_sdk_to_client
             request_class = TokenClassificationQuery
+        elif task == TaskType.TEXTTOTEXT:
+            from rubrix.sdk.api.text_to_text import _get_dataset_data
+
+            map_fn = self._text2text_sdk_to_client
+            request_class = Text2TextQuery
         else:
             raise ValueError(
-                f"Sorry, load method is only allowed with token and text classification"
+                f"Sorry, load method not supported for the '{task}' task. Supported tasks: "
+                f"{[TaskType.TEXTCLASSIFICATION, TaskType.TOKENCLASSIFICATION, TaskType.TEXTTOTEXT]}"
```

Comment:
Could we do some kind of refactoring like:

```python
...
map_fn, request_class = _TASK_CONFIG[task]
```

defining the `_TASK_CONFIG` dictionary like:

```python
_TASK_CONFIG = {
  TaskType.TEXTCLASSIFICATION: (self._text_classification_sdk_to_client, TextClassificationQuery),
  TaskType.TOKENCLASSIFICATION: (self._token_classification_sdk_to_client, TokenClassificationQuery),
  TaskType.TEXTTOTEXT: (self._text2text_sdk_to_client, Text2TextQuery),
}
```